common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -6,37 +6,48 @@
+
+To the autopilot, vibration noise looks like any other disturbance (e.g. wind) that the autopilot must compensate for in order to control the aircraft. ArduPilot uses a :ref:`low-pass <INS_GYRO_FILTER>` software filter to remove much of this remaining vibration noise, however, filtering has an unwanted side-effect - it removes *everything* including attitudinal and control information about the aircraft. Thus, while aggressive filtering can eliminate noise it can also reduce the control and responsiveness of the aircraft. This problem gets particularly acute on helicopters with very high levels of vibration noise at harmonics of the rotor speed and small, high powered Copters with low-levels of natural mechanical damping that require greater control.
 
 For multicopters, virtually all vibrations originate from the motors and, importantly, quite a lot is known about this noise source as the majority of it is linked to the motor rotational frequency.  For helicopters, the vibrations are linked to the rotor speed.  It is thus theoretically possible to construct a software filter that targets *just* this noise, leaving all of the useful gyro information alone. This is what a *notch filter* does - it targets a narrow band of frequencies.
 
-ArduPilot has support for two different notch filters - a :ref:`static notch filter<common-imu-notch-filtering-static-notch>` that can be set at a fixed frequency and a dynamic notch filter that can be targeted at a range linked to the motor rotational frequency for multicopter motors or the rotor speed for helicopters.
+ArduPilot has support for two different notch filters - a single :ref:`static notch filter<common-imu-notch-filtering-static-notch>` that can be set at a fixed frequency and dynamic notch filters that can be targeted at a range linked to the motor rotational frequency for multicopter motors or the rotor speed for helicopters. The dynamic notch filters are actually a set of several filters placed at harmonics of the main filter frequency.
 
 With the introduction of dynamic notch filtering, the need for static notch filtering is reduced, although it can be useful in targeting particular resonant frequencies (e.g. of the frame). We will therefore look at dynamic filtering first.
 
+Key to the dynamic notch filter operation is control of its center frequency. There are five methods that can be used for doing this:
+
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 0. Dynamic notch disabled. The center frequency is fixed and is static. Often used in Traditional Helicopters with external governors for rotor speed, either incorporated in the ESC or separate for ICE motors.
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1. (Default) Throttle position based, where the frequency at mid-throttle is determined by analysis of logs, and then variation of throttle position above this is used to track the increase in noise frequency.
```

Comment:
Actually disabled is the default

Reply:
yeah, we need to add the INS_HNTCH_ENABLE param at the top...will do